web/docs/Backend Starters/Decoupled Drupal/local-development.md

```diff
@@ -0,0 +1,89 @@
+# Local Development
+
+## Before You Begin
+
+The example local environment configurations outlined below are based on [Lando](https://lando.dev/). To install Lando, follow the [Lando installation instructions](https://docs.lando.dev/getting-started/installation.html).
+
+Depending on your role, you may only need a local backend environment, or you may instead prefer a local environment that can run both the back-end and front-end. The following sections outline the steps to set up a local environment for each of these use cases.
+
+## Backend Only Setup
+
+If you are creating a local environment for your WordPress backend only, you can use the default Pantheon recipe provided by Lando. For more information, see the [Getting Started section of the Lando Pantheon Plugin documentation](https://docs.lando.dev/pantheon/getting-started.html).
+
+## Combined Frontend and Backend Setup
+
+- Create `.lando.yml` in the root of the project.
+- Copy the following config into `.lando.yml`
+
+```
+name: %SITE_NAME%
+recipe: pantheon
+config:
+  framework: drupal9
+  site: %SITE_NAME%
+  id: %PANTHEON_SITE_ID%
+  xdebug: false
+
+proxy:
+  frontend:
+    - %SITE_NAME%-fe.lndo.site:%PORT-NUMBER%
+
+excludes:
+  - frontend/node_modules
+  - node_modules
+
+services:
+  appserver:
+    xdebug: false
+    config:
+      php: .vscode/php.ini
```

Comment:
They won't have access to the php.ini file referenced here. Let's remove the xdebug related items for now, and we can create additional xdebug related documentation later.